package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@indigov/zendesk-api",
-  "version": "0.0.101",
+  "version": "0.0.107",
   "description": "The official Indigov wrapper for the Zendesk API",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "repository": "github:indigov-us/zendesk-api",
   "scripts": {
     "test": "jest --verbose",
     "sandbox": "ts-node -r dotenv/config src/sandbox.ts",
-    "build": "tsc",
-    "prepublish": "tsc"
+    "build": "npx tsc",
+    "prepublish": "npx tsc"
```

Comment:
why? we're already including typescript as a dev dependency...

Reply:
If you don't use npx then it looks for the system `tsc` first where this will make sure it's looking for the dev dependency.